schedule/yast/yast2_ncurses_gnome.yaml

```diff
@@ -1,50 +1,51 @@
 ---
 name:           yast2_ncurses_gnome
 description:    >
-    Test for yast2 UI, ncurses only. Running on created gnome images
-    which provides both text console for ncurses UI tests as well as
-    the gnome environment for the GUI tests.
+  Test for yast2 UI, ncurses only. Running on created gnome images
+  which provides both text console for ncurses UI tests as well as
+  the gnome environment for the GUI tests.
 schedule:
-    - "{{pre_boot_to_desktop}}"
-    - boot/boot_to_desktop
-    - console/prepare_test_data
-    - console/consoletest_setup
-    - "{{yast_modules}}"
-    - console/coredump_collect
+  - "{{pre_boot_to_desktop}}"
+  - boot/boot_to_desktop
+  - console/prepare_test_data
+  - console/consoletest_setup
+  - console/yast2_i
```

Comment:
could you please put this module inside the conditions repeating it for the 4 archs? As I could check in other PR the order matters and having conditions for all architecture it is more clear to use the same order and repeat the modules. Also it is easy to understand why yast2_i is outside {{yast2_modules}}. In further work we can optimize, but for now I'd like that we follow this suggestion. You don't need extra verification for that, just put it in the original order.

Reply:
Sorry, but nothing indicates that this module plays any role in the order.

Reply:
not this one, but it has been proved that is confusing for this tests to have some common part when you are still using all the the arch to do the conditional, basically it is an invitation for the next one to fail and spend time searching for common denominator in this kind of tests. But still if you want to keep it it is fine for me.

Reply:
In my point of you, if this causes any problem, it would mean a bug. Fixing the order to make it work, it would be not ideal as it would hide the problem. Maybe i dont understand something. But i really believe that is more than ok even as it is